### DISCUSSION TITLE
q-select default dropdown open

### DISCUSSION THREAD

**Discussion by AaditT** [quasarframework/quasar#16054] [General - Components / Directives / etc]
Hey, is there a prop/attribute or way to call the q-select component such that the dropdown options are always open by default?

**Comment by suleimanas** [comment]
Hi. Yes using showPopup() method
`
<q-select ref="input_select" v-model="model" :options="options" label="Standard"></q-select>`

```
const input_select= ref();
    
onMounted(()=>{
      input_select.value.showPopup()
});
```
